test/checks/aria/required-children.js

```diff
@@ -32,8 +32,8 @@ describe('aria-required-children', function() {
 			var shadowRoot = target.attachShadow({ mode: 'open' });
 			shadowRoot.innerHTML = '<p>Nothing here.</p>';
 
-			var tree = (axe._tree = axe.utils.getFlattenedTree(fixture));
-			var virtualTarget = axe.utils.getNodeFromTree(tree[0], target);
+			axe._tree = axe.utils.getFlattenedTree(fixture);
+			var virtualTarget = axe.utils.getNodeFromTree(target);
```

Comment:
Can we do something to make this not look like an unnecessary assignment? Maybe create a simple `setupFlatTree()` method or something? Looking for feedback here. This is just to hide an unintuitive setup.

```js
axe.utils.setupFlatTree = function setupFlatTree (node) {
  axe._tree = axe.utils.getFlattenedTree(node);
}
```

Reply:
Would this public method be used outside of tests?

Reply:
Probably best put in `test/testutils.js`